internal/pb/export/export.proto

```diff
@@ -41,21 +41,30 @@ message TemporaryExposureKeyExport {
   // Information about signatures
   repeated SignatureInfo signature_infos = 6;
 
-  // The TemporaryExposureKeys themselves
+  // The TemporaryExposureKeys for initial release of keys.
+  // Keys should be included in this list for initial release,
+  // whereas revised or revoked keys should go in revised_keys.
   repeated TemporaryExposureKey keys = 7;
+
+  // TemporaryExposureKeys that have changed status.
+  // Keys should be included in this list if they have changed status
+  // or have been revoked.
+  repeated TemporaryExposureKey revised_keys = 8;
 }
 
 message SignatureInfo {
-  // No longer need to set these. Old code that still does will be fine
-  // as these were largely noops anyway.
+  // These fields are no longer needed (confirmed with Apple)
```

Comment:
I'd revert this comment. That these are confirmed with Apple is sorta implied all throughout.

Reply:
done